pipestat_example/looper_config_pipestat.yaml

```diff
@@ -0,0 +1,32 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+  project: ./pipeline/pipeline_interface_project.yaml
+pipestat:
+  project:
+    project_name: test
+    sample_name: frog_1
+    schema_path: ./pipeline/pipestat_output_schema.yaml
```

Comment:
should we change this key to "output_schema" ?

Reply:
Left as `schema_path` to be consistent with pipestat.